### PR TITLE
added a def, fixed styling, included if __name__...

### PR DESCRIPTION
**Pull request by nvega23**
added a def named mult for multiples, fixed the styling to be more readable, included an if __name__, and included assert tests.

### CODE REVIEW

File: mult_of_three_and_five.py [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+    for x in range(num1):
+        if (x % 3 == 0 or x % 5 == 0):
+            total += x
+
+    return total
+
+
+if __name__ == "__main__":
+    total = mult(1000)
+    print(total)
```
Comment (kswang2400) [inline mult_of_three_and_five.py]: nit: you can delete these two lines since you have the asserts already :)

File: mult_of_three_and_five.py [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
-for x in range(1000):
-    if (x % 3 == 0 or x % 5 == 0):
-        total = total + x
-        print(total)
+# If we list all the natural numbers below 10 that are multiples of 3 or 5,
+# we get 3, 5, 6 and 9. The sum of these multiples is 23.
+
+# Find the sum of all the multiples of 3 or 5 below 1000.
+
+def mult(num1):
```
Comment (kswang2400) [inline mult_of_three_and_five.py]: nit: function naming can be more descriptive :)

**Review by kswang2400 — APPROVED** [review]
nice!

**Comment by nvega23** [comment]
for the commit multiples of three and fives lines 16, 17 were removed and the function was renamed to mults_three_five

**Review by kswang2400 — APPROVED** [review]
nice!

**Pull request merged**